mitodl/mitxpro
# chore: remove external_marketing_url from course and program run

#### Pre-Flight checklist


#### What are the relevant tickets?
#2636 

#### What's this PR do?
- Removes unused external_marketing_url field from CourseRun model
- Stops a data migration that depends upon the removed field

#### How should this be manually tested?
- The external marketing url related should work fine in course details page
- Check course feasibility e.g. enrollment and course details pages
- Check that `/api/courses`, `/api/programs`, `/api/products` are working fine

#### Any background context you want to provide?
We migrated `external_marketing_url` field in https://github.com/mitodl/mitxpro/pull/2628 from CourseRun to course pages, Now the field in the Course/Program Run is useless. So, that's removed in this PR.
